Content.Shared/RetractableItemAction/RetractableItemActionComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.RetractableItemAction;
+
+/// <summary>
+/// Used for storing an unremovable item within an action and summoning it into your hand on use.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(fieldDeltas: true), Access(typeof(RetractableItemActionSystem))]
```

Comment:
```suggestion
[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, Access(typeof(RetractableItemActionSystem))]
```
you are not using them